src/main/java/sirius/biz/storage/layer2/L3Uplink.java

```diff
@@ -359,16 +359,32 @@ private boolean deleteHandler(VirtualFile file) {
         return false;
     }
 
-    private boolean moveHandler(VirtualFile file, VirtualFile newParent) {
+    private boolean canFastMoveHandler(VirtualFile file, VirtualFile newParent) {
```

Comment:
javadoc ?: can move file into new parent directory if they belong both to the same space

Reply:
well, its a private method as simply provides the implementation as defined by MutableVirtualFile.withCanFastMoveHandler